### DISCUSSION TITLE
node js taking forever to install ...and it was a globally installed module.. sometime its still downloading the packages rather than install the globally installed module on my pc

### DISCUSSION THREAD

**Discussion by firdausakbarr** [nodejs/help#5038] [Q&A]
### Description of the problem

node js taking forever to install ...and it was a globally installed module.. sometime its still downloading the packages rather than install the globally installed module on my pc

**Comment by firdausakbarr** [comment]
![NPM STUCK](https://user-images.githubusercontent.com/70452536/215487374-6b87ad3e-03dd-408e-a720-59d64de4efb5.png)

**Comment by aduh95** [comment]
It looks like you meant npm, not Node.js. Those are two separate projects, this repo is about Node.js development, not npm.

To address the issue you seem to have, did you try to use something that's not npm to see if it would fit your needs better? E.g. [pnpm](https://pnpm.io) might be a good candidate.

**Reply by firdausakbarr** [reply]
Pnpm or yarn just ram the same issues... they cant install from global installed packages
